lib/archive.go

```diff
@@ -27,18 +27,23 @@ import (
 	"io"
 	"io/ioutil"
 	"os"
-	"path/filepath"
+	"path"
 	"regexp"
 	"sort"
 	"strings"
 	"time"
 )
 
-var homeDirRE = regexp.MustCompile(`^/(Users|home|Documents and Settings)/(?:[^/]+)`)
+var volumeRE = regexp.MustCompile(`^([a-zA-Z]):(.*)`)
+var sharedRE = regexp.MustCompile(`^//([^/]+)`) // matches a shared folder in Windows after backslack replacement. i.e //VMBOXSVR/k6/script.js
```

Comment:
I'm a bit concerned that this regex could match valid *nix paths like `//etc/hosts` (same as `/etc/hosts`). I see that `path.Clean()` (which removes multiple slashes) is called, but only after `NormalizeAndAnonymizePath()`. 

Why don't we call `path.Clean()`/`filepath.Clean()` at the beginning, before we start any checks or manipulations of the path?